### ISSUE TITLE
\lipsum seems to have stopped issuing a \par

### ISSUE THREAD

**Issue by FrankMittelbach** [PhelypeOleinik/lipsum#12]
According to the documentation (and past behavior) `\lipsum` issued a `\par` and `\lipsum*` a space and the other way around if the option `nopar` is used. Neither seems to be the case any more:
```
\documentclass{article}
\usepackage
%    [nopar]
    {lipsum}

\begin{document}
\small
\lipsum [1][1-4]¶
\lipsum*[2][1-2]
\lipsum [3][1-2]¶
\lipsum [4][1-3]¶
\ldots
\end{document}
```
Breaks the TLC3 documentation on that package :-(

**Comment by PhelypeOleinik** [comment]
Maybe I'm missing something, but the output with v2.2 (oldest version that supports a second optional argument that I could find) and v2.7 seems to be the same (TL 2018 and 2022).  This:
```latex
\documentclass{article}
\usepackage
%    [nopar]
    {lipsum}

\begin{document}
\fmtversion --- \csname ver@lipsum.sty\endcsname\par

\small
\lipsum [1][1-4]¶
\lipsum*[2][1-3]
\lipsum [3][1-2]¶
\lipsum [4][1-3]¶
\ldots
\end{document}
```

with TL 2018 gives:

> ![test1](https://user-images.githubusercontent.com/12678598/170987688-46f2e83d-66c6-4068-9ded-cda3cf193faf.png)

and with LT 2022:

> ![test2](https://user-images.githubusercontent.com/12678598/170987706-a8b18c41-ef8c-4a96-b202-a01bdce0ab91.png)

The only difference is a line break on line 7 of the output PDF, but that's because on v2.7 `lipsum` uses latin hyphenation patterns (`ul-lam-cor-per` with english hyphenation vs `ul-la-m-cor-per` with latin).

**Comment by FrankMittelbach** [comment]
maybe you are missing nothing and it is broken since 2018, but the documentation still states that `\lipssum` and `\lipssum*` differ in whether or not they issue a `\par` or a `space` and that `nopar` swaps the behavior.

It it was certainly the case when I wrote the documentation for it (which is unfortunately that long ago :-( ). Looks to me as if the star form doesn't have any effect at all any more and perhaps that's since 2018.

Also, the corresponding package kantlipsum has exactly that behavior (i.e., if you replace `\lipsum` with `\kant`).

**Comment by PhelypeOleinik** [comment]
That's true for `\lipsum` with one optional argument.  With two optional arguments it runs in "sentence mode", outputting a range of sentences, and ending with a `\par` would not allow `hello \lipsum[1][1] world`, so I think it's the right default.

You can do `\setlipsum{sentence-after=\par}` to make it add a `\par` at the end.
